core/templates.yml

```diff
@@ -60,12 +60,63 @@ keys:
         type: str
         shotgun_entity_type: HumanUser
         shotgun_field_name: login
+
     custom_key_example:
         type: str
         validate_hook: 'templatekey_custom'
```

Comment:
You could also *just* define the value_from_str_hook and leave validate_hook and str_from_value_hook blank and the code will fallback on the default functionality.

Reply:
Will do that in ingest.yml, although the hook is also calling the parent.
We can define the keys in ingest.yml. And maybe basic examples of these templates. But these will mostly be sourced from show's preference files for sgtk_config. As per the client tech specs.